content/news/2024-07-01-GCC2024/index.md

```diff
@@ -0,0 +1,127 @@
+The Freiburg Galaxy team contributed by providing training for:
+
+* [BioConda & BioContainers](https://galaxyproject.org/events/gcc2024/training/bioconda) by Björn Grüning
+* [Machine Learning](https://galaxyproject.org/events/gcc2024/training/machine-learning) by Anup Kumar *et. al.*
+* [Galaxy Backend Development](https://galaxyproject.org/events/gcc2024/training/backend-dev) by David Lopez *et. al.*
+* [Alphafold in Galaxy](https://galaxyproject.org/events/gcc2024/training/alphafold) by Björn Grüning *et. al.*
+* [Pathogen Detection from (Direct Nanopore) Sequencing Data using Galaxy](https://galaxyproject.org/events/gcc2024/training/microbiome-analysis) by Engy Nasr
+* [CSS Deep-Dive](https://galaxyproject.org/events/gcc2024/training/frontend-dev) by Laila Los *et. al.*
+* [Collecting, Organizing, and Broadcasting Galaxy Tools for a Dedicated Scientific Community (using the Galaxy CoDex)](https://galaxyproject.org/events/gcc2024/training/frontend-dev) by Paul Zierep
+
+## Panel Discussion
+
+## Birds of a Feather (BoF)
+
+## CoFest
+
+## Social Events
+
+### Conference Dinner and Mendel Museum
+
+### City Tours and Villa Tugendhat
+
+On Wednesday, we had a delightful day exploring the [Villa Tugendhat](https://www.tugendhat.eu/en/) and [the city of Brno](https://www.gotobrno.cz/en/explore-brno/). A total of 31 people, divided into two groups, visited the Villa Tugendhat, while 66 people in two groups toured the city.
+
+One Villa tour group took the tram, and another group enjoyed a walk despite the warm weather. The Villa Tugendhat visit was particularly special due to its remarkable modernity for its time. Designed by German architect Ludwig Mies van der Rohe, the villa was built in just two years by the wealthy and influential Jewish Czech Tugendhat family. It is recognized as a UNESCO World Heritage Site.
+
+During the city tour, we explored the main sights of Brno's city center with guided tours organized by the Brno Tourist Information Centre. We visited the Old Town Hall, St. Peter's Cathedral, as well as the Freedom and Market Squares. The experience was enjoyable and provided a rich insight into Brno's history and culture.
+
+### Conference Nights Activities
+
+Beyond the enriching presentations and discussions, conference nights were filled with fun and camaraderie. Groups of attendees ventured out to the Bouldering Rock Climbing Hall, testing their climbing skills and bonding over shared challenges. Others enjoyed friendly competitions at local billiards halls.
+
+![GCC2024 Billiards](gcc2024billiards.jpg)
+
+Dinner outings were a highlight, with groups exploring Brno’s culinary scene, sampling delicious Mexican and Indian cuisine. The evenings often concluded with open-air talks, where participants relaxed and enjoyed local beers and [Kofola](https://en.wikipedia.org/wiki/Kofola) at various pubs and bars, further strengthening connections and creating memorable experiences.
```

Comment:
I would also mention the local cuisine: Schnitzel! @bgruening

Reply:
I couldnot skip the Kofola :D :D <3 <3, I don't know about the Schnitzel story, seems like I missed some of the dinner gatherings :D